src/main/java/com/coveros/selenified/utilities/Property.java

```diff
@@ -49,13 +49,15 @@
 public class Property {
 
     private static final String PROXY_ISNT_SET = "Proxy isn't set";
+    private static final double wait = 5;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS00115)